libs/azure-iot-middleware-freertos/libraries/azure-sdk-for-c/sdk/src/azure/iot/az_iot_adu_client.c

```diff
@@ -205,6 +205,16 @@ AZ_NODISCARD az_result az_iot_adu_client_get_agent_state_payload(
       ref_json_writer, AZ_SPAN_FROM_STR(AZ_IOT_ADU_CLIENT_AGENT_PROPERTY_NAME_MODEL)));
   _az_RETURN_IF_FAILED(az_json_writer_append_string(ref_json_writer, device_properties->model));
 
+  for (uint32_t customPropertyIndex = 0;
```

Comment:
real nit for code convention
```suggestion
  for (uint32_t custom_property_index = 0;
```